app/controllers/posts_controller.rb

```diff
@@ -1,18 +1,20 @@
 class PostsController < ApplicationController
   before_action :logged_in_user, only: [:create]
 
-  def show
-    @post = Post.find(params[:id])
-  end
-
   def new
     @post = Post.new
   end
 
+  def show
+    @post = Post.find(params[:id])
+    @comments = @post.comments.order('created_at ASC')
+    @comment = Comment.new
```

Comment:
Worth using `@post.comments.build` here so the `post_id` on the comment is already set?  I'm not too familiar with how posts and comments are tied together, so disregard if I'm completely off base.